runhouse/resources/hardware/utils.py

```diff
@@ -88,6 +88,21 @@ class ServerConnectionType(str, Enum):
     AWS_SSM = "aws_ssm"
 
 
+def _load_cluster_config() -> Dict:
```

Comment:
EnvServlets on the cluster will never have to load from file, or depend on it. They can simply call this, and they'll get the config from an already initialized ClusterServlet.